src/app/components/algorithms/algorithm-list/algorithm-list.component.ts

```diff
@@ -67,7 +69,6 @@ export class AlgorithmListComponent implements OnInit {
   }
 
   onAddElement(): void {
-    const params: any = {};
     const dialogRef = this.utilService.createDialog(
       AddAlgorithmDialogComponent,
       {
```

Comment:
in line 81: const algorithmDto: **any** = {
See linter warning

Reply:
if (dialogResult) {
        const algorithmDto: any = {
          name: dialogResult.name,
          computationModel: dialogResult.computationModel,
        };

This is followed by this conditional check
        if (algorithmDto.computationModel === 'QUANTUM') {
          algorithmDto.quantumComputationModel =
            dialogResult.quantumComputationModel;
        }
So could not directly give the parameters inside the usage 
this.algorithmService.createAlgorithm({ body: algorithmDto }).subscribe(.........
as done in other files

Reply:
see my last commit - I used a global variable wich is either `QuantumAlgorithmDto` or `ClassicAlgorithmDto` to remove the any